tests/fixtures/excerpts_CRLF/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# force tests to keep their Windows EOL
+* text eol=crlf
```

Comment:
Should there be a non-root editorconfig here?

Reply:
we can have multiple editorconfig as well? I'll them tomorrow :D